pithos/plugins/notify.py

```diff
@@ -77,3 +80,47 @@ def on_disable(self):
         if self._shutdown_handler:
             self._app.disconnect(self._shutdown_handler)
             self._shutdown_handler = 0
+
+
+class NotifyPluginPrefsDialog(Gtk.Dialog):
+    __gtype_name__ = 'NotifyPluginPrefsDialog'
+
+    def __init__(self, window, settings):
+        super().__init__(use_header_bar=1)
+        self.set_title(_('Notification Preferences'))
+        self.set_default_size(300, -1)
+        self.set_resizable(False)
+        self.connect('delete-event', self.on_close)
+
+        self.pithos = window
+        self.settings = settings
+        self.show_album = literal_eval(self.settings['data']) if self.settings['data'] else False
```

Comment:
Let's not do `literal_eval`. A simple `bool()` to get it and `str()` to store it is plenty.

Reply:
The issue is that Boolean strings don't convert to bool types easily in Python.

```python
>>> bool('True')
True
>>> bool('False')
True
```

Reply:
I can try switching it to string comparison.

Reply:
Ah. Yes a simple string comparison is good.